posix-mapper/src/main/webapp/WEB-INF/web.xml

```diff
@@ -0,0 +1,150 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!DOCTYPE web-app
+        PUBLIC "-//Sun Microsystems, Inc.//DTD Web Application 2.3//EN"
+        "http://java.sun.com/j2ee/dtds/web-app_2_3.dtd">
+
+<web-app>
+
+    <display-name>posix-mapper</display-name>
+
+    <servlet>
+        <servlet-name>logControl</servlet-name>
+        <servlet-class>ca.nrc.cadc.log.LogControlServlet</servlet-class>
+        <init-param>
+            <param-name>logLevel</param-name>
+            <param-value>info</param-value>
+        </init-param>
+        <init-param>
+            <param-name>logLevelPackages</param-name>
+            <param-value>
+                ca.nrc.cadc.rest
+                ca.nrc.cadc.util
+                ca.nrc.cadc.vosi
+                ca.nrc.cadc.db
+                org.hibernate
+                org.opencadc.posix
+            </param-value>
+        </init-param>
+        <init-param>
+            <param-name>logControlProperties</param-name>
+            <param-value>posix-logControl.properties</param-value>
+        </init-param>
+        <load-on-startup>1</load-on-startup>
+    </servlet>
+
+    <servlet>
+        <servlet-name>PosixUserManagementServlet</servlet-name>
+        <servlet-class>ca.nrc.cadc.rest.RestServlet</servlet-class>
+        <init-param>
+            <param-name>augmentSubject</param-name>
+            <param-value>false</param-value>
+        </init-param>
+        <init-param>
+            <param-name>init</param-name>
+            <param-value>org.opencadc.posix.mapper.web.PosixInitAction</param-value>
+        </init-param>
+        <init-param>
```

Comment:
I don't understand why there is a post action? doesn't get action create a mapping if necessary and return it? That was the idea (for both user and group)